.github/workflows/pr-merge.yml

```
@@ -74,7 +74,7 @@ jobs:
         run: node ./.github/scripts/commit.js
       - name: get new npm version
         id: package-version
-        uses: martinbeentjes/npm-get-version-action@master
+        run: echo "current-version=$(node -p "require('./packages/sdk/package.json').version")" >> $GITHUB_OUTPUT
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:87: Double quote to prevent globbing and word splitting

(shellcheck)